Tests/Marketplace/marketplace_services.py

```diff
@@ -538,6 +538,8 @@ def _parse_pack_dependencies(first_level_dependencies, dependencies_metadata_dic
         parsed_result = {}
 
         for dependency_id, dependency_data in dependencies_metadata_dict.items():
+            if dependency_id in Metadata.EXCLUDED_DEPENDENCIES:
```

Comment:
Didn't you mean this?
```suggestion
            if dependency_id in dependency_data.get(Metadata.EXCLUDED_DEPENDENCIES, []):
```